EIDR-C/index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <style type="text/css">
+    iframe {
+      width: 100%;
+      height: 537px;
+      border: none;
+    }
+  </style>
+</head>
+  <!-- Test -->
+  <iframe></iframe>
+  <script type="text/javascript" src="https://developer.bsvecosystem.net/sdk/api/BSVE.API.js"></script>
+  <script>
+  var IFRAME_URL = "https://eidr-connect.eha.io/";
```

Comment:
I think the slash at the end could be a problem when checking the iframe message origin.